### PR TITLE
update dwb plugin pkg readme

### PR DESCRIPTION
**Pull request by SteveMacenski**
Handles #1316 just giving some more flavor to the readme for DWB.

Also makes some assumptions on the merging of #1377 which I don't think is a controversial topic

### CODE REVIEW

File: nav2_dwb_controller/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ an implementation of both DWA and Trajectory Rollout.
 ## Migrating to ROS 2
 
 Rather than continue with 3 overlapping implementations of DWA, the DWB planner
-was chosen to use as the standard local planner.
+was chosen to use as the standard controller plugin to start.
```
Comment (mkhansen-intel) [inline nav2_dwb_controller/README.md]:
```suggestion
was chosen to use as the default controller plugin.
```

File: nav2_dwb_controller/README.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ The advantages of DWB were:
 
 For the most part, the DWB codebase is unchanged, other than what was required for ROS2.
 
-The main architectural change is due to the replacement of `MoveBase` by the `FollowPath` action server.
+The main architectural change is due to the replacement of `MoveBase` by the `FollowPath` action server that uses this plugin.
```
Comment (mkhansen-intel) [inline nav2_dwb_controller/README.md]:
```suggestion
The main architectural change is due to the replacement of `MoveBase` by the `FollowPath` action server, called the controller_server, which uses this DWB plugin.
```

File: nav2_dwb_controller/README.md [hunk 3]
```diff
@@ -27,10 +27,10 @@ The advantages of DWB were:
...
 
 ![ROS 1 DWB Structure](./images/DWB_Structure_Simplified.svg "ROS 1 DWB Structure")
 
 ## New local planner interface
 
-See `nav2_core` `LocalPlanner` interface. They are loaded into the `nav2_controller_server` to compute control commands from requests to the ROS2 action server.
+See `nav2_core` `Controller` interface. They are loaded into the `nav2_controller_server` to compute control commands from requests to the ROS2 action server.
```
Comment (mkhansen-intel) [inline nav2_dwb_controller/README.md]:
```suggestion
See `nav2_core` `Controller` interface, which defines an interface for the controller plugins. These plugins are loaded into the `nav2_controller_server` to compute control commands from requests to the ROS2 action server.
```

**Comment by SteveMacenski** [comment]
All good comments. Will do when in office.

**Comment by SteveMacenski** [comment]
@mkhansen-intel please take another look

File: nav2_dwb_controller/README.md [hunk 4]
```diff
@@ -27,10 +27,10 @@ The advantages of DWB were:
 
 For the most part, the DWB codebase is unchanged, other than what was required for ROS2.
 
-The main architectural change is due to the replacement of `MoveBase` by the `FollowPath` action server.
+The main architectural change is due to the replacement of `MoveBase` by the `FollowPath` action server, called the controller_server, which uses this DWB plugin.
```
Comment (mjeronimo) [inline nav2_dwb_controller/README.md]: Perhaps splitting hairs, but MoveBase was replaced by the BtNavigator, which interfaces to the various action servers - ComputePathToPose, FollowPath, recoveries, etc. - using action clients.

Comment (SteveMacenski) [reply nav2_dwb_controller/README.md]: This is in context of the nav2 dwb controller readme, so I think it makes sense that this was pulled out of movebase, but what you said probably makes more sense

Comment (SteveMacenski) [reply nav2_dwb_controller/README.md]: @mjeronimo I think that entire sentence doesn't provide much value. I'm just deleting it, its in a section discussing DWB changes for ROS2 and none of that should be important for a plugin to understand or care about.

**Pull request merged**